TelCo.ColorCoder/ColorMap.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+    /// <summary>
+    /// This class provides the color coding and mapping of pair number to color and color to pair number.
+    /// </summary>
+    class ColorMap
```

Comment:
Class Name is ColorMap but its prints Mannual as well ?

Look for Class  Cohesion  rules

Reply:
Created a new class, called ColorPair which prints Color pair Manual.
And ColorMap class will have functionality of mapping colors based on Pair number and vice versa.